packages/terra-popup/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-popup",
   "main": "lib/Popup.js",
-  "version": "6.71.0",
+  "version": "6.72.0",
```

Comment:
The question same as above - if fixes are patches. Please disregard if they are supposed to be minors

Reply:
[59e4ab1](https://github.com/cerner/terra-framework/pull/1832/commits/59e4ab1088afe400e2915ca480cb7a13ac1a3964)